frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/components/ComponentPortlet.java

```diff
@@ -111,6 +116,10 @@
 )
 public class ComponentPortlet extends FossologyAwarePortlet {
 
+    private static final String QUERY_PARAMS_FOSSOLOGY = "?mod=showjobs&upload=";
+
+    private static final String URL_PREFIX = "https://";
```

Comment:
It would be good, if we can extract Protocol `http/https` from the FOSSology URL , using `getScheme()` , also  port can be used if present using `getPort()` on `URI` Object

Reply:
Added the logic to compute the protocol and port from the fossology rest api link.